fedimg/utils.py

```diff
@@ -154,8 +154,7 @@ def get_image_name_from_image(image_url, virt_type='', region='', respin='0',
     file_name = get_file_name_image(image_url)
     build_name = file_name.replace('.raw.xz', '')
 
-    return '-'.join(
-        [x for x in [build_name, virt_type, region, volume_type, respin] if x])
+    return build_name
```

Comment:
If we are doing this then we need to change it so that the function doesn't require all of those arguments (that are no longer used and update any calls to this function).